integration-tests/gradle/gradle6-test/build.gradle.kts

```diff
@@ -15,20 +15,31 @@
  * limitations under the License.
  */
 
-plugins {
-    id("realm-lint")
-}
-
 // Explicitly adding the plugin to the classpath as it makes it easier to control the version
 // centrally (don't need version in the 'plugins' block). Further, snapshots are not published with
 // marker interface so would need to be added to the classpath manually anyway.
 buildscript {
-    dependencies {
-        classpath("io.realm.kotlin:gradle-plugin:${Realm.version}")
-    }
+        extra["realmVersion"] = file("${rootProject.rootDir.absolutePath}/../../../buildSrc/src/main/kotlin/Config.kt")
```

Comment:
Yes. It we use `buildSrc` to gain access to `Config.kt` it is problematic to use Kotlin and AGP versions that are different from the ones in `Config.kt` because `buildSrc` applies a number of plugins and that will fix the versions.